CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 ### Changed
 - Docker - Fix lumber version in `Dockerfile`.
+- Technical - database analyzer refacto - separate each database analyzer in its own file and add errors.
```

Comment:
```suggestion
- Technical - Split database analysis logic in 2 separated files (SQL and NoSQL).
```

Reply:
Let me know your advice about the analyzer's names above before changing here.

Reply:
I answered for the analyzers names, up to you know ;)

Reply:
okay